modules/openglqt/CMakeLists.txt

```diff
@@ -58,6 +61,9 @@ target_link_libraries(inviwo-module-openglqt PRIVATE
     Qt${QT_VERSION_MAJOR}::Svg
     Qt${QT_VERSION_MAJOR}::OpenGLWidgets
```

Comment:
Does this work for Qt5 even though there is no find_package for it?

Reply:
The openglwidgets is new in qt6 and the qopenglwidget it moved to it...